src/Elm/Compiler/Error.elm

```diff
@@ -16,13 +16,16 @@ module Elm.Compiler.Error exposing
 
 -}
 
+import Array
+import Elm.Data.FileContents exposing (FileContents)
 import Elm.Data.FilePath exposing (FilePath)
 import Elm.Data.Located exposing (Located)
 import Elm.Data.ModuleName exposing (ModuleName)
 import Elm.Data.Type as Type exposing (Type)
 import Elm.Data.Type.ToString as TypeToString
 import Elm.Data.VarName exposing (VarName)
 import Json.Decode as JD
+import List.Extra
```

Comment:
this probably needs to get deleted for the build to pass

Reply:
```suggestion
```